template/11-monetization.md

```diff
@@ -1,6 +1,13 @@
 # Business Model
 
-*Expected operating Costs*
+### Expected operating cost
+The operating costs of our app are proportional to the number of users. The choice of Supabase allows us to avoid vendor lock-in if it becomes too costly to pay for the database and storage. The pricing page of Supabase shows us that 25$ per month is sufficient for 100'000 monthly active users. Then the cost per user increases, but thanks to the fact that Supabase can be deployed on our own hardware, we can keep the cost down. For example, if we decide to deploy Supabase on a virtual private server, we estimate that it will cost only 0.00015$ per month per monthly active user. We clearly see that these operating costs are small.
 
-*Revenue Streams*
+### Revenue stream
+The revenue stream of our app comes from mutliples sources.  
+First, we can collect some group behavior data from our users. For example, which categories of events are the most successful, or what's the best location for an event to be successful. With these insights on our users, we can introduce an optional monthly subscription of $5 for the associations which want to improve their chances to organize a successful event.  
+Second, we can develop themed versions of the app for special occasions. For example a week-long event on a big area with a lot of events in parallel, like CES, could ask us to create a special app for the occasion that would allow the participants to have an overview of all the events, join some of them and find them easily on the map. This would occasionally bring us a big income, probably about 10'000$ per mission.  
+Third, we can monetize some services in our application, which allows the associations to have easy integration of all the things that come with the organization of an event. For example, a ticketing system can easily be added to our app. The user could buy a ticket for a paid event without needing another app or website. The associations would have some management interface with the participant to allow easy verification, refunds, and more. We just take a small fee on the ticket price, for example, 5%.
 
```

Comment:
Some more ideas for monetisation: 
Nathan: We could have some partnership with some of the activity places such as gyms, swimming pools, soccer and tennis grounds such that they could pay us a deductible and we would promote them on our app. 

(Note these can be considered as ideas for further monetisation as well as considered for future extrapolation of the app)
Later on, introduce a premium version for users which would allow them to have discounts on their tickets and benefit from early bird prices for various events, possibility of keeping the user data private and hidden such that it is not taken into account during our data analysis.

Have a photos section where photos from different events will be posted. How to smartly filter people who do not give permissions to use their photos by minimally affecting the quality of photos or the filtering process? People who do not give permission to use their photos would have to declare so either on their profile or in a permission section in the 'join event' screen; put a specific sticker (available to collect from the organizers on the day of the event or prior) on their arm or their chest. Our machine learning algorithm, when it would detect a sticker on a participant's body would automatically blur out their face! The stickers which would be used by these people and would be a requirement of all events could be our own personalized logo stickers and we could charge some fee while selling them to the event organizers.